src/emhass/web_server.py

```diff
@@ -297,18 +325,18 @@ def action_call(action_name):
     else:
         injection_dict = None
     
-    if args.addon == 1:
+    if hass_url:
```

Comment:
The web server needs to be generic for the add-on but also for the standalone docker installation method. That argument `args.addon` was a way to know which installation method was used and how we access the configuration file.